ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/texteditor/TextEditorPresenter.java

```diff
@@ -292,6 +293,7 @@ public void onDocumentReceived(final String content) {
                     @Override
                     public void onContentInitialized() {
                         document.setCursorPosition(currentCursor);
+                        generalEventBus.fireEvent(new FileContentUpdatedEvent(document.getFile()));
```

Comment:
FileContentUpdateEvent and FileContentUpdatedEvent are different event. When we fire FileContentUpdateEvent we need not know when this task will be completed. But for debugger I need know when content completely rendered, and only after that I can render debug line.